eng/pipelines/templates/jobs/integration.yml

```diff
@@ -8,16 +8,16 @@ jobs:
   - download: current
     displayName: Download $(PipelineArtifactName)_packed
     artifact: $(PipelineArtifactName)_packed
-    
+
   - template: /eng/pipelines/templates/steps/publish-to-dev-feed.yml
     parameters:
       PathToArtifacts: $(Pipeline.Workspace)/$(PipelineArtifactName)_packed
       Registry: https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-js/npm/registry/
       ${{ if and(eq(variables['Build.SourceBranch'], 'refs/heads/main'), eq(variables['Build.Reason'], 'IndividualCI')) }}:
-        Tag: latest
-      ${{ else }}:
         Tag: dev
-      
+      ${{ else }}:
```

Comment:
1. Please keep contributing.md up to date with the state of the world.  Can you include updates there with this PR?  Specifically document what "dev" and "pre" and "latest" mean for us and our partners please.

What does pre mean?  What happens to the latest tag after this change?  Does any existing workflow or links break?

Reply:
Added